packages/brick-kit/src/internal/evaluate.ts

```diff
@@ -262,6 +263,11 @@ export function evaluate(
         return {
           getUrl: getUrlBySegueFactory(app, segues),
         };
+      case "THEME":
+        return {
+          curTheme: getTheme(),
+          isDark: getTheme() === DARK_THEME,
+        };
```

Comment:
属性做成方法比较好，你这里实际每次都要计算两次 `getTheme()`，虽然用户可能只要其中一个。

```suggestion
      case "THEME":
        return {
          getTheme,
          // 这个 dark 和 dark-v2 有歧义，建议先不加吧，先让用户用 `THEME.getTheme() === "dark-v2"`。
          // isDark: getTheme() === DARK_THEME,
        };
```